test/functional/wallet_keypool.py

```diff
@@ -127,7 +127,10 @@ def run_test(self):
         nodes[0].keypoolrefill(3)
 
         # test walletpassphrase timeout
-        time.sleep(1.1)
+        # CScheduler relies on condition_variable::wait_until() which is slightly
+        # less accurate than libevent's event trigger. We'll give it 2
```

Comment:
Seems weird to mention libevent here as a point of reference since it's in the past after this change is applied. I would put this in the commit message and just say that it's not accurate enough here.